pylib/cqlshlib/tracing.py

```diff
@@ -73,8 +73,6 @@ def make_trace_rows(trace):
     if trace.duration:
         finished_at = (datetime_from_utc_to_local(trace.started_at) + trace.duration)
         rows.append(['Request complete', str(finished_at), trace.coordinator, total_micro_seconds(trace.duration), trace.client])
-    else:
-        finished_at = trace.duration = "--"
```

Comment:
@bschoening  why have you removed this?

Reply:
The value of finished_at the else is unused in that scope.  This was either a pyflake or pylint nit.